sqlglot/parser.py

```diff
@@ -553,6 +553,15 @@ class Parser(metaclass=_Parser):
         TokenType.PROPERTIES: lambda self: self._parse_wrapped_csv(self._parse_property),
     }
 
+    OPTION_PARSERS = {
+        TokenType.BLOCKCOMPRESSION: lambda self: self._parse_blockcompression(),
```

Comment:
we shouldn't need token types for these, we should make it so that OPTION_PARSERS takes in text, otherwise you have to always add token types

so change to 
{
"BLOCKCOMPRESSION": ...
} 
and delete all the token types you've added

we could possibly refactor property parsers to do the same thing but don't necessarily have to do that